src/redux/book/book.js

```diff
@@ -1,26 +1,29 @@
 import { createAction } from '@reduxjs/toolkit/src/createAction';
+import { createReducer } from "@reduxjs/toolkit/src/createReducer";
 import { nanoid } from '@reduxjs/toolkit';
-
-const INITIAL_STATE = [];
-
-const bookReducer = (state = INITIAL_STATE, actions) => {
-  if (actions.type === 'ADD_BOOK') {
-    return [...state, actions.payload];
-  } if (actions.type === 'REMOVE_BOOK') {
-    return [state.filter((book) => book !== actions.payload.id)];
+const ADD_BOOK = createAction('bookstore/books/ADD_BOOK', (book) => {
+  return {
+    payload: {
+      id: nanoid(),
+      title: book.title,
+      author: book.author,
+      completed: false,
+    }
```

Comment:
- [ ] This is an action creator as the previous reviewer said you should create actions separately as variables and use them in the actions creator like the following example 
```javascript
const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK'
and then use this variable inside the action creators 
```